db/migrate/20150925033708_add_column_to_candidates.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToCandidates < ActiveRecord::Migration
```

Comment:
Yeah - this was suppose to be "AddColumn'Featured'ToCandidates" but when I ran rails g migration I typed it in wrong. The code below it is correct